src/services/__tests__/catalogue.test.ts

```diff
@@ -36,7 +36,7 @@ describe("CATALOGUE service", () => {
     })
 
     it("returns data", async () => {
-      const fetched = await fetchModels("audi")
+      const fetched = await fetchModels({ makeKey: "audi" })
```

Comment:
I liked the unnamed version of this. not sure if it's personal preference, but when there is only 1 argument and it's a required one, I'd not go for named attributes

Reply:
also, this breaking change is pretty hefty. we'll need to update pretty much all existing calls. I don't see the value in exercising through that

Reply:
I prefer to have them aligned and don't have to think if I need to pass an object or not. This also makes extending API calls easy. Plus you know immediately what parameter you pass does. It's fairly similar to always forcing parens by prettier in that regard in my opinion.
I'll take care of the update. typecheck task will do a good job guiding me through it so I'm not concerned here

Reply:
thanks, let's roll with it